javatests/arcs/android/storage/database/DatabaseImplTest.kt

```diff
@@ -175,10 +175,14 @@ class DatabaseImplTest {
 
     @Test
     fun createEntityStorageKeyId_createsNewIds() = runBlockingTest {
+        val creationTimestamp = 99L
+        val expirationTimestamp = 999L
```

Comment:
You could make these constants, e.g. `CREATION_TIMESTAMP` (put them at the bottom of the file). Then you wouldn't have to redeclare them every time.

Reply:
done, thanks!